packages/app/obojobo-repository/shared/components/repository-nav.scss

```diff
@@ -1,5 +1,7 @@
 @import '../../client/css/defaults';
 
+$poppin-pink: #af1b5c;
```

Comment:
Since this color is used in two places, remove the declarations from the individual stylesheets and put them in a single location used by everything that requires the color.

Also name more descriptively, e.g. `$color-notification`.